tests/lang-rust.yaml

```diff
@@ -4,8 +4,8 @@
   assert:
   - status == 0
   - stdout.indexOf("rustc") != -1
-- desc: it should have properly configured rustup
-  command: [rustup show]
+- desc: it should have properly functioning rustup
+  command: [rustup show && rustup update && rustup toolchain install nightly && cargo new /tmp/foo] # Testing out `cargo` at the end is necessary to know that rustup didn't mess things up
```

Comment:
`rustup toolchain install nightly`
just double checking, do we indeed want to install nightly?

Reply:
The bug was observable only after installing another toolchain. So, yes, to have a test that reproduces the problem it is necessary to install an additional toolchain such as `beta` or `nightly`.  `nightly` is relevant I think because it is the most used after `stable`.